app/src/main/res/values/strings.xml

```diff
@@ -181,10 +181,10 @@
     <string name="english">English</string>
     <string name="french">French</string>
     <string name="german">German</string>
-    <string name="italian">Italian</string>
-    <string name="marathi">Marathi</string>
     <string name="hebrew">Hebrew</string>
+    <string name="italian">Italian</string>
     <string name="macedonian">Macedonian</string>
+    <string name="marathi">Marathi</string>
```

Comment:
No Norwegian Bokmål?

Reply:
had it forgotten tbh, it doesn't exist in settings preferences